state_of_css.yml

```
@@ -356,21 +554,21 @@ translations:
   - key: features_others.combinators
     t: Комбінатори
   - key: features_others.combinators.description
-    t: Які з цих CSS-комбінаторів ви використовували?
+    t: Які серед цих CSS-селекторів для комбінацій ви використовували?
```

Не знаю, чому оригінал такий дивний, але мова саме про комбінатори
```suggestion
    t: Які серед цих комбінаторів селекторів CSS ви використовували?
```